libs/java/server_common/pom.xml

```diff
@@ -31,6 +31,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.pulsar</groupId>
```

Comment:
What if we want to use a different messaging service such as RabbitMQ, ActiveMQ, or something else already in use by whoever using Athenz? Maybe we should make it configurable?

Reply:
This pulsar implementation implements:
`PulsarFactory<T> implements ChangePublisherFactory<T>, ChangeSubscriberFactory<T>`

We chose to add specific pub/sub implementation, pulsar, here.
Additional MQ implementation can be added easily.

Reply:
Got it